src/enc/brotli_bit_stream.rs

```diff
@@ -1347,6 +1347,20 @@ impl<
             distance_histograms_size: 0,
         }
     }
+}
+
+impl<
+        Alloc: alloc::Allocator<u8>
+            + alloc::Allocator<u32>
+            + alloc::Allocator<HistogramLiteral>
+            + alloc::Allocator<HistogramCommand>
+            + alloc::Allocator<HistogramDistance>,
+    > MetaBlockSplit<Alloc>
+{
+    pub fn new() -> Self {
+        Self::default()
```

Comment:
why does this one call default and the one below call new. We should probably decide on a direction and stick to it

Reply:
i think new should always call default (and maybe eventually obsolete new) - simply because new without params is not standard (not a trait), whereas default is.  Fixed here. I will clean them up more later